media/media.css

```diff
@@ -147,10 +147,24 @@ body {
   margin-bottom: 0.125rem;
 }
 
+.extraItem-select-path {
+  display: flex;
+  width: 120%;
+  justify-content: space-between;
+}
+
 vscode-text-field.extraItemInput {
   width: 120%;
 }
 
+.folders-img {
+  height: 1rem;
+  position: relative;
+  top: 1.6rem;
+  z-index: 9;
+  margin-right: 0.25rem;
```

Comment:
Why not work with `right` here?

The problem is that `position: relative` just places it relative to its usually position. I think having it absolutely positioned would work better, or?

Reply:
But that's rather due to the relative container. Have an appropriate wrapper that is `position: relative` and it works fine.